src-java/projectfloodlight/openflowj/build.gradle

```diff
@@ -45,15 +44,7 @@ clean.dependsOn(cleanSources)
 task generateSources(type: Exec, dependsOn: cloneLoxigenRepo) {
     outputs.dir('loxigen/loxi_output/openflowj')
     workingDir 'loxigen'
-
-    if (System.getenv('TRAVIS') == 'true') {
```

Comment:
if you removed it you should remove `export TRAVIS=true` from `open-kilda/.github/workflows/unittest.yml`

Reply:
fixed